app/models/ability.rb

```diff
@@ -46,9 +46,13 @@ def not_signed_in
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can :show, User
     unless CONFIG['authentication']['ichain']['enabled']
-      can [:show, :create], Registration do |registration|
+      can :show, Registration do |registration|
         registration.new_record?
       end
+
+      can [:new, :create], Registration do |registration|
+        registration.conference.registration_open? && registration.new_record?
```

Comment:
What about a signed_in user? Are we going to limit access to new and create actions, unless registration is open?